static/app/views/profiling/content.tsx

```diff
@@ -243,4 +265,15 @@ const ActionBar = styled('div')`
   margin-bottom: ${space(2)};
 `;
 
+// TODO: another simple primitive that can easily be <Grid columns={2} />
+const PanelsGrid = styled('div')`
+  display: grid;
+  grid-template-columns: 50% 50%;
+  gap: ${space(2)};
+
+  @media (max-width: ${p => p.theme.breakpoints.small}) {
+    grid-template-columns: 100%;
+  }
+`;
+
```

Comment:
another example of a simple layout component we can have...

`<Grid columns={2} />` while taking into consideration breakpoints etc.